test/arrays/userAPI/arrayAPItest.chpl

```diff
@@ -17,6 +17,7 @@ proc testArrayAPI1D(lbl, X: [], sliceDom, reindexDom) {
   writeln();
   // Test simple queries
   writeln("size is: ", X.size);
+
```

Comment:
Was this blank line added intentionally?  (and there's another one at 118)

Reply:
Yes, because doing so preserved the line numbers reported in the .good files and didn't require me to mess with them in that way.